samples/cpp/stereo_calib.cpp

```diff
@@ -345,14 +379,19 @@ int main(int argc, char** argv)
     Size boardSize;
     string imagelistfn;
     bool showRectified;
-    cv::CommandLineParser parser(argc, argv, "{w|9|}{h|6|}{s|1.0|}{nr||}{help||}{@input|stereo_calib.xml|}");
+    cv::CommandLineParser parser(argc, argv, "{w|9|}{h|6|}{t|chessboard|}{s|1.0|}{ms|0.5|}{ad|0|}{adf|None|}{nr||}{help||}{@input|stereo_calib.xml|}");
     if (parser.has("help"))
         return print_help(argv);
     showRectified = !parser.has("nr");
     imagelistfn = samples::findFile(parser.get<string>("@input"));
     boardSize.width = parser.get<int>("w");
     boardSize.height = parser.get<int>("h");
+    string type = parser.get<string>("t");
     float squareSize = parser.get<float>("s");
+    float markerSize = parser.get<float>("ms");
+    int arucoDict = parser.get<int>("ad");
+    string arucoDictFile = parser.get<string>("adf");
```

Comment:
The same as for interactive calibration and calibration.cpp. Users do not know dict codes, name selection is needed.